app/docker/helpers/containerHelper.js

```diff
@@ -54,5 +115,122 @@ angular.module('portainer.docker')
     return config;
   };
 
+  helper.preparePortBindings = function(portBindings) {
+    const bindings = {};
+    portBindings.forEach(function (portBinding) {
+      if (portBinding.containerPort) {
```

Comment:
Extract code out of `if` block, and replace this `if` condition with
```
if (!portBinding.containerPort) {
  return;
}
```

Reply:
Makes sense. I'm not sure why I did it that way. Fixed.